app/code/Magento/PageCache/view/frontend/web/js/page-cache.js

```diff
@@ -45,31 +44,31 @@ define([
          * @param {jQuery} element - Comment holder
          */
         (function lookup(element) {
-            var iframeHostName;
-
-            // prevent cross origin iframe content reading
-            if ($(element).prop('tagName') === 'IFRAME') {
-                iframeHostName = $('<a>').prop('href', $(element).prop('src'))
-                    .prop('hostname');
-
-                if (window.location.hostname !== iframeHostName) {
-                    return [];
-                }
-            }
-
             /**
              * Rewrite jQuery contents().
              *
              * @param {jQuery} elem
              */
             contents = function (elem) {
                 return $.map(elem, function (el) {
+                    var iframeHostName;
+
+                    // prevent cross origin iframe content reading
+                    if ($(element).prop('tagName') === 'IFRAME') {
+                        iframeHostName = $('<a>').prop('href', $(element).prop('src'))
+                            .prop('hostname');
+
+                        if (window.location.hostname !== iframeHostName) {
+                            return [];
+                        }
+                    }
+
```

Comment:
looks like it should be for `el`
```suggestion
                    var iframeHostName;

                    // prevent cross origin iframe content reading
                    if ($(el).prop('tagName') === 'IFRAME') {
                        iframeHostName = $('<a>').prop('href', $(el).prop('src'))
                            .prop('hostname');

                        if (window.location.hostname !== iframeHostName) {
                            return [];
                        }
                    }

```

Reply:
Updated as your suggestion